vision/google/cloud/vision/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Google Inc.
+# Copyright 2017, Google Inc. All rights reserved.
```

Comment:
nit: copyright is from the original publication date, it is not the last modified date.